deploy/Dockerfile

```diff
@@ -51,18 +36,11 @@ RUN \
 
 FROM scratch
 COPY --from=0 /src/out/executor /kaniko/executor
-COPY --from=0 /usr/local/bin/docker-credential-gcr /kaniko/docker-credential-gcr
-COPY --from=0 /usr/local/bin/docker-credential-ecr-login /kaniko/docker-credential-ecr-login
-COPY --from=0 /usr/local/bin/docker-credential-acr-env /kaniko/docker-credential-acr-env
-COPY --from=certs /ca-certificates.crt /kaniko/ssl/certs/
-COPY --from=0 /kaniko/.docker /kaniko/.docker
```

Comment:
I'm tempted to say that we should leave this alone for a release and encourage folks to try `slim` because the NEXT release will do precisely this.  WDYT?

Reply:
I'd be okay with that. I can just revert all the Dockerfile changes in this PR. 👍